### PR TITLE
Add more commands

### PR DESCRIPTION
**Pull request by drashna**
Adds additional commands for stuff new(er) to the docs.

### CODE REVIEW

**Review by fauxpark — CHANGES_REQUESTED** [review]
Should also add a command for the udev rules.

**Comment by drashna** [comment]
> Should also add a command for the udev rules.

You mean https://docs.qmk.fm/#/faq_build?id=linux-udev-rules ?

**Comment by fauxpark** [comment]
Yep. It comes up often enough.

**Comment by drashna** [comment]
Added.

File: utils.js [hunk 1]
```diff
@@ -6,7 +6,7 @@ const helpmessage =  [
   },
   {
     name: "**FAQs**",
-    value: "faq, faq-build, faq-debug, faq-keymap, glossary"
+    value: "faq, faq-build, faq-debug, faq-keymap, squeeze-avr, glossary"
   },
   {
     name: "**Building, Flashing, Drivers**",
```
Comment (fauxpark) [inline utils.js:12]: Udev should go under this section maybe?

Comment (drashna) [reply utils.js:12]: Added

**Review by fauxpark — APPROVED** [review]

**Review by ridingqwerty — APPROVED** [review]
Looks good

**Pull request merged**